crashes/index.html

```diff
@@ -61,6 +62,9 @@
 var gData = undefined;
 var gMap = {};
 
+// When populated, this is a map activity_date -> channel -> version
+var gLatestVersions = getLatestVersions();
```

Comment:
The first seven dates don't work as they rely on data being present a week before. I suppose I can make that safe, but it's always been that way. I've filed #284 for this.